### PR TITLE
Solved bug related to quiztimer and departure date

### PR DESCRIPTION
**Pull request by sridharjajoo**
1) The timer in the quiz now pauses when the dialog box appears and resumes on cancel.(#126)
2)The departure date cannot be before the current date.(#130)

### CODE REVIEW

**Comment by yatna** [comment]
@sridharjajoo can you separate the task into 2 commits , one for each task (use git reset soft , then add 2 commits )

**Comment by sridharjajoo** [comment]
@yatna I have made a commit for the bug in the timer. If I make another commit then there will be two commits for a single PR. So should I make the commit after merging of above branch or include the "DEPARTURE DATE"  commit in this PR itself?

**Comment by yatna** [comment]
Yes it is perfectly fine to have numerous commits in one PR. 
For keeping the code neat we prefer 1 commit per task, here this PR fixes 2 different issues , hence it should have 2 commits.

**Comment by sridharjajoo** [comment]
@yatna ThankYou . Your guidance is really helpful. I have made the commit.Kindly review it!

**Comment by yatna** [comment]
@sridharjajoo Glad to hear that.

**Comment by yatna** [comment]
@chhavip Looks fine to me.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java [hunk 1]
```diff
@@ -411,19 +411,34 @@ public void onClick(View v) {
...
+
+                    else if(arrivall < departurel)
+                    {
+                        Toast.makeText(getApplicationContext(),"Arrival date cannot be before the Departure date.",Toast.LENGTH_SHORT).show();
+                    }
+
+                    else if(arrivall>=departurel)
+
+                    {
+                        //Log.v("","PAPA " + arrivall);
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java]: Remove this log.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java [hunk 2]
```diff
@@ -411,19 +411,34 @@ public void onClick(View v) {
...
-                    if(arrivall>=departurel) {
+                    if(departurel < curr_datel){
+
+                        Toast.makeText(getApplicationContext(),"Departure date cannot be less than current date",Toast.LENGTH_SHORT).show();
+
+                    }
+
+                    else if(arrivall < departurel)
+                    {
+                        Toast.makeText(getApplicationContext(),"Arrival date cannot be before the Departure date.",Toast.LENGTH_SHORT).show();
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java]: Put values in strings.xml and do not hardcode.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java [hunk 3]
```diff
@@ -411,19 +411,34 @@ public void onClick(View v) {
...
+                    }
+
+                    else if(arrivall>=departurel)
+
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java]: Remove extra lines and follow the same { } pattern as done elsewhere in the code. Of opening in the same line.

**Comment by chhavip** [comment]
The last commit will need removal. Only the first two should be submitted.

**Comment by sridharjajoo** [comment]
@chhavip I have reset the commits and included the formatting changes as well as the fix related to arrival and departure time in a new commit so that there are 2 commits.Hope it is fine.

**Comment by sridharjajoo** [comment]
@chhavip I have made the formatting changes

**Comment by chhavip** [comment]
Good work! Merging. Can we try to keep two issues in two separate PRs from next time?

**Pull request merged**